sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubMessageSerializerTest.java

```diff
@@ -181,56 +179,60 @@ public void deserializeEventHubProperties() {
+            final Date createdAtAsDate  = new Date(1569275540L);
+            final String[] partitionIds = new String[]{ "1", "foo", "bar", "baz" };
 
-        final Map<String, Object> values = new HashMap<>();
-        values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, partitionIds);
+            final Map<String, Object> values = new HashMap<>();
+            values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, partitionIds);
 
-        final AmqpValue amqpValue = new AmqpValue(values);
+            final AmqpValue amqpValue = new AmqpValue(values);
 
-        final Message message = Proton.message();
-        message.setBody(amqpValue);
+            final Message message = Proton.message();
+            message.setBody(amqpValue);
 
-        // Act
-        serializer.deserialize(message, EventHubProperties.class);
+            // Act
+            serializer.deserialize(message, EventHubProperties.class);
+        });
     }
 
     /**
      * Verify that it throws if the value in the map is null.
      */
-    @Test(expected = AzureException.class)
+    @Test
     public void throwsWhenNullValueInResponse() {
-        // Arrange
-        final String eventHubName = "event-hub-name-test";
-        final Date createdAtAsDate  = new Date(1569275540L);
+        assertThrows(AzureException.class, () -> {
+            // Arrange
+            final String eventHubName = "event-hub-name-test";
+            final Date createdAtAsDate  = new Date(1569275540L);
 
-        final Map<String, Object> values = new HashMap<>();
-        values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, null);
+            final Map<String, Object> values = new HashMap<>();
+            values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, null);
 
-        final AmqpValue amqpValue = new AmqpValue(values);
+            final AmqpValue amqpValue = new AmqpValue(values);
 
-        final Message message = Proton.message();
-        message.setBody(amqpValue);
+            final Message message = Proton.message();
+            message.setBody(amqpValue);
 
-        // Act
-        serializer.deserialize(message, EventHubProperties.class);
+            // Act
+            serializer.deserialize(message, EventHubProperties.class);
```

Comment:
Only this line should be in the `assertThrows(() -> {})`, to scope down where we expect the error to be.

Reply:
thanks for the review. changes are incorporated